### ISSUE TITLE
Algorithms

### ISSUE THREAD

**Issue by aniketsharma00411** [aniketsharma00411/algorithmsUse#1]
Add any Algorithm which is not already present in its respective Programming Language directory.

If you don't find your favorite programming language's directory, feel free to create it.

**Comment by hpnightowl** [comment]
can i take this issue?

**Comment by SubhamChoudhury** [comment]
Can I contribute to this issue?

**Comment by saurabhatram** [comment]
Can I help ?

**Comment by ghost** [comment]
can I contribute to this ?

**Comment by sakshi0461** [comment]
can I contribute to this ?

**Comment by sauravrajleaf** [comment]
Can i contribute to this ?

**Comment by aniketsharma00411** [comment]
Anyone can contribute any algorithm. Feel free to add anything you want.

**Comment by saurabhatram** [comment]
Changes made @aniketsharma00411  PULL request[#6 ](https://github.com/aniketsharma00411/algorithmsUse/pull/6)

**Comment by hackbansu** [comment]
Adding binary search in java.

**Comment by hackbansu** [comment]
Added #24 Binary search in Java.

**Comment by hydra-sketch** [comment]
can I get a chance to contribute this?

**Comment by zainbinfurqan** [comment]
#33 i have added quick sort algo in js @aniketsharma00411

**Comment by hpnightowl** [comment]
#36 added sjf algo

**Comment by princekalra** [comment]
can i contribute on this

**Comment by Heisenberg-737** [comment]
Can I contribute to this?

**Comment by Renu-code** [comment]
Can I add topological sort in this repo?

**Comment by princekalra** [comment]
i want to work  on this please assign it to me

**Comment by joey2031** [comment]
Hello, I would like to add the Cocktail Sort in C++. Thanks!

**Comment by ayushontop** [comment]
I would like to contribute .

**Comment by aniketsharma00411** [comment]
> I would like to contribute .

@ayushontop First read the CONTRIBUTING.md file and follow instructions there.

**Comment by chawlapalak** [comment]
Hello @aniketsharma00411 , I would like to work on this.

**Comment by aniketsharma00411** [comment]
> Hello @aniketsharma00411 , I would like to work on this.

Yeah, sure. Read the CONTRIBUTING.md file and open PR accordingly. If you have any further questions feel free to ask.

**Comment by anishsingh935** [comment]
Hi please confirm me that, do we have to add algorithms in repository ??  or this issue is for something else  ??

**Comment by aniketsharma00411** [comment]
> Hi please confirm me that, do we have to add algorithms in repository ?? or this issue is for something else ??

You can add any algorithm in the repo which is not already present or improve the previously added ones.

**Comment by anishsingh935** [comment]
ok got it thanks for suggesting .

**Comment by anishsingh935** [comment]
I am adding  more new and efficient algorithm that I have .
